Source/WebKit/UIProcess/API/glib/WebKitWebsiteDataManager.cpp

```diff
@@ -118,6 +119,8 @@ struct _WebKitWebsiteDataManagerPrivate {
     GUniquePtr<char> swRegistrationsDirectory;
     GUniquePtr<char> domCacheDirectory;
 #endif
+
+    guint64 perOriginStorageQuota { 0 };
```

Comment:
```suggestion
    guint64 perOriginStorageQuota;
```

(The priv struct is already zero-initialized.)